pkg/kor/argorollouts.go

```diff
@@ -0,0 +1,214 @@
+package kor
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/yonahd/kor/pkg/clusterconfig"
+	"github.com/yonahd/kor/pkg/filters"
+)
+
+func GetUnusedArgoRollouts(clientsetinterface clusterconfig.ClientInterface, namespace string, filterOpts *filters.Options) ResourceDiff {
```

Comment:
We need to create cmd/kor/argorollouts.go so users can select this one specifically 
Also missing pkg/kor/multi.go - allow finding your new resource in a comma-separated query along other resources.